app/assets/stylesheets/show.scss

```diff
@@ -8,6 +8,7 @@ $form-border-color: #ee7600;
 .vote-buttons,
 .rover-name,
 .rover-description {
+  color: #dd7e28;
```

Comment:
Color literals like `#dd7e28` should only be used in variable declarations; they should be referred to via variable everywhere else.